src/commands/init.ts

```diff
@@ -0,0 +1,79 @@
+import commander from "commander";
+import dotenv = require("dotenv");
+import * as fs from "fs";
+import yaml from "js-yaml";
+import * as os from "os";
+import * as env from "../config";
+import { logger } from "../logger";
+import { Command } from "./command";
+
+const defaultFileLocation = os.homedir() + "/.spk-config.yml";
```

Comment:
This is the default filepath, similar to kubeconfig, after they run init, we save the file at `~/.spk-config.yml` so that they don't have to keep passing us path to the config file for every command after init. This has been moved to another file